packages/backend-for-frontend/test/api/consumerDelegationRouter/getConsumerDelegatedEservices.test.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import { generateId } from "pagopa-interop-models";
+import { generateToken } from "pagopa-interop-commons-test";
+import { authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { api, services } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+import { getMockBffApiCompactEService } from "../../mockUtils.js";
+
+describe("API GET /consumers/delegations/eservices", () => {
+  const defaultQuery = {
+    offset: 0,
+    limit: 5,
+    delegatorId: generateId(),
+  };
+  const mockCompactEServices = {
+    results: [
+      getMockBffApiCompactEService(),
+      getMockBffApiCompactEService(),
+      getMockBffApiCompactEService(),
+    ],
+    pagination: {
+      offset: defaultQuery.offset,
+      limit: defaultQuery.limit,
+      totalCount: 3,
+    },
+  };
+
+  beforeEach(() => {
+    services.delegationService.getConsumerDelegatedEservices = vi
+      .fn()
+      .mockResolvedValue(mockCompactEServices);
+  });
+
+  const makeRequest = async (token: string, query: object = defaultQuery) =>
```

Comment:
```suggestion
  const makeRequest = async (token: string, query: typeof defaultQuery = defaultQuery) =>
```
(to match conventions in other PRs – same applies to query params in other files)

Reply:
Updated across all tests